### DISCUSSION TITLE
show 'uploading' indicator on dock icon

### DISCUSSION THREAD

**Discussion by Offbeatmammal** [iterate-ch/cyberduck#15614] [Q&A] [answered]
often when I set some files to upload, I change away to the browser or editor to work on something else, and don't know when uploads are finished (I don't like lots of notifications)
would be nice to have an option to animate or show an indicator on the duck when there are transfers happening and revert to normal icon when finished.

**Comment by dkocher** [comment, marked as answer]
You can set the [hidden property](https://docs.cyberduck.io/cyberduck/preferences/#hidden-configuration-options) `queue.dock.badge` to `true` using 

```
defaults write ~/Library/Preferences/ch.sudo.cyberduck.plist queue.dock.badge true
```

to add a indicator to the dock application icon showing the nunmber or currently running transfers.

**Reply by dkocher** [reply]
This is [documented](https://docs.cyberduck.io/cyberduck/transfer/#badge-dock-icon).

**Reply by Offbeatmammal** [reply]
I've just done a clean install of Cyberduck, and this seems to have stopped working? Did it both with the way above and the documented suggestion, and confirmed with a defaults read, but no badge appearing (it was before the re-install). I miss my little red dot!

**Reply by Offbeatmammal** [reply]
d'oh, I forgot to allow notifications for Cyberduck on macOS System Settings. All good again
